pkg/controller/controller.go

```diff
@@ -90,6 +90,11 @@ type csiProvisioner struct {
 	config               *rest.Config
 }
 
+type driverState struct {
+	driverName string
+	accessibilityConstraintCapability bool
```

Comment:
a more clear name could be "supportsAccessiblity"

Reply:
or you could potentially make this generic and have an enum of all the different capabilities